examples/flutter_counter/lib/counter/counter.dart

```diff
@@ -1,2 +1,3 @@
 export 'cubit/counter_cubit.dart';
-export 'view/view.dart';
```

Comment:
I believe this was correct as is. Why change this?

Reply:
Tested this today with Flutter 3.3.4 and getting error "URI doesn't exist"
![obraz](https://user-images.githubusercontent.com/2189840/195991050-dd6e1377-d449-4052-abff-7287ee94ed60.png)

Reply:
There is a view file: https://github.com/felangel/bloc/blob/master/examples/flutter_counter/lib/counter/view/view.dart

Reply:
Take a look at Docs:
https://bloclibrary.dev/#/fluttercountertutorial?id=counter-view
it says:
Let's create **lib/counter/view/counter_view.dart**
not lib/counter/view/view.dart

Reply:
Good catch! We should update the barrel section in the docs right after to include creating a barrel for view as well. Feel free to open a PR to fix the docs. The code is fine as is.